aspnetcore/host-and-deploy/windows-service/samples/7.x/WebAppServiceSample/Services/ServiceA.cs

```diff
@@ -1,34 +1,27 @@
-using System;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.Extensions.Hosting;
-using Microsoft.Extensions.Logging;
+namespace SampleApp.Services;
```

Comment:
I'm not sure why the diff file is so awful, I just used the new namespace declaration and removed implicit using's.

Reply:
Yeah, git always does this going to file-scoped namespaces. Turning off whitespace in the diff sorts it out mostly.